packages/sheets-numfmt/src/index.ts

```diff
@@ -16,6 +16,7 @@
 
 export { UniverSheetsNumfmtPlugin } from './numfmt-plugin';
 export { getPatternPreview } from './utils/pattern';
+export { NumfmtMenuController } from './controllers/numfmt.menu.controller';
```

Comment:
Currency symbols should be exported as well so users know what currencies are available.